### DISCUSSION TITLE
if keda scales a deployment to 0 and therefore no endpoints are available, does ingress complain?

### DISCUSSION THREAD

**Discussion by Loki-Afro** [kedacore/keda#4974] [Q&A / Need Help]
in a "Normal" kubernetes
ingress -> service -> deployment -> pods

if an deployment gets scaled to 0, ingress throws errors that there are no endpoints available.
```
W0913 15:13:14.663060       7 controller.go:1100] Error obtaining Endpoints for Service "<namespace>/<svc>": no object matching key "<namespace>/<svc>" in local store
```

if keda scales a deployment to 0 and therefore no endpoints are available either, does ingress complain as well?
If that is the case, how can this be prevented?

**Comment by tomkerkhove** [comment]
If Ingress fails about 0 instances, then it will complain if you do it with or without KEDA. We won't make a difference.
